src/vanilla/Templates/MethodTemplate.cshtml

```diff
@@ -9,7 +9,13 @@
 @{
 string methodName = Model.Name.ToPythonCase();
 string defKeyword = AsyncMode? "async def" : "def";
+string trace_decorator = "";
 }
+@if (Model.AddTracingDecorators)
+{
+trace_decorator = AsyncMode? "@distributed_trace_async" : "@distributed_trace";
+}
+    @(trace_decorator)
```

Comment:
I would not do a variable, and do directly the "if Model.AddTracingDecorators" to avoid an empty line if --trace is not used

Reply:
Interestingly this does not actually create a new line.